tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = py{38,39}-django{22}-drf{311,312}
           py{38,39,310,311,312}-django{32}-drf{312,313,314}
           py{38,39,310,311,312}-django{42}-drf{314}
-          py{310,311,312}-django{51}-drf{314}
+          py{310,311,312}-django{51}-drf{314,315}
```

Comment:
This version should also be tested with dj 4.2 lts

Reply:
ok I am adding it for django 4.2 also